include/internal/libspdm_common_lib.h

```diff
@@ -343,6 +343,21 @@ typedef struct {
 } libspdm_msg_log_t;
 #endif /* LIBSPDM_ENABLE_MSG_LOG */
 
+#if LIBSPDM_FIPS_MODE
+typedef struct {
+    /**
+     * Tested algo flag: 0 represents that the algo is not tested.
+     * See LIBSPDM_FIPS_SELF_TEST_xxx;
+     **/
+    uint32_t tested_algo;
+    /**
+     * Flag for the result of run algo self_test, 0 represents the result is failed.
+     * See LIBSPDM_FIPS_SELF_TEST_xxx;
+     **/
+    uint32_t self_test_result;
+} libspdm_crypto_context;
```

Comment:
The name "libspdm_crypto_context" is kind of misleading - "crypto" is too big a scope, it's really just FIPS selftest status.

Reply:
Yes.  I have changed the `libspdm_crypto_context` to `libspdm_fips_selftest_context`.